compass/landice/tests/ismip6_forcing/shelf_collapse/process_shelf_collapse.py

```diff
@@ -0,0 +1,281 @@
+                             f"ISMIP6 only for CCSM4-RCP85, "
+                             f"CESM2-WACCM-SSP585,HadGEM2-ES-RCP85, "
+                             f"UKESM1-0-LL-SSP585 model&scenarios. Please "
+                             f"set up the correct config options in "
+                             f"the config file.")
+
+        input_file = self._files[period_endyear][model][scenario] \
+                     [res_ismip6][0]
+        self.add_input_file(filename=mali_mesh_file,
+                            target=os.path.join(base_path_mali,
+                                                mali_mesh_file))
+        self.add_input_file(filename=os.path.basename(input_file),
+                            target=os.path.join(base_path_ismip6,
+                                                input_file))
+        self.add_output_file(filename=f"{mali_mesh_name}_"
+                                      f"{os.path.basename(input_file)}")
+
+    def run(self):
+        """
+        Run this step of the test case
+        """
+        logger = self.logger
+        config = self.config
+        section = config["ismip6_ais"]
+        period_endyear = section.get("period_endyear")
+        output_base_path = section.get("output_base_path")
+        model = section.get("model")
+        scenario = section.get("scenario")
+        res_ismip6 = section.get("res_ismip6")
+        mali_mesh_name = section.get("mali_mesh_name")
+        mali_mesh_file = section.get("mali_mesh_file")
+
+        # we always want neareststod for the remapping method because we want
+        # a value between 0 and 1 per mesh point.
+        method_remap = "neareststod"
+
+        # interpolate and rename the data
+        # ismip6 input files
+        input_file = self._files[period_endyear][model][scenario] \
+                     [res_ismip6][0]
+
+        logger.info(f"!---Start processing the file---!")
+        logger.info(f"processing the input file "
+                    f"'{os.path.basename(input_file)}'")
+
+        # temporary file name.
+        remapped_file_temp = f"remapped.nc"
+
+        # remap the input forcing file.
+        logger.info(f"Calling the remapping function...")
+        self.remap_ismip6_shelf_mask_to_mali_vars(os.path.basename(input_file),
+                                                  remapped_file_temp,
+                                                  mali_mesh_name,
+                                                  mali_mesh_file,
+                                                  method_remap)
+
+        output_file = f"{mali_mesh_name}_{os.path.basename(input_file)}"
+
+        # rename the ismip6 variables to MALI variables
+        logger.info(f"Renaming the ismip6 variables to "
+                    f"mali variable names...")
+        self.rename_ismip6_shelf_mask_to_mali_vars(remapped_file_temp,
+                                                   output_file)
+
+        # round up/down the mask values to 1/0
```

Comment:
I believe the error has to do with the `-O` flag to `ncap2`. If the `-O` flag is replaced with `-A` I'm able to execute the problem line without any problem (e.g.): 
```
ncap2 -A -s "where(calvingMask>=0.5) calvingMask=1" $FILE_NAME.nc
```
I've never really used `ncap2` so I'm not sure if there's a problem with switching from `-O` to `-A`. I wonder if it would make more sense to switch to `xarray` to do the rounding. Just feels a little weird to call a command line utility to do NetCDF arithmetic within `python`, but I could be missing something. Or maybe we can remove these lines all together since the interpolation method is hardcoded as `neareststod`.